core/src/main/java9/com/linecorp/armeria/internal/Java9RequestContextAwareCompletableFuture.java

```diff
@@ -25,271 +25,247 @@
 import java.util.function.Function;
 import java.util.function.Supplier;
 
+import com.linecorp.armeria.common.RequestContext;
 import com.linecorp.armeria.common.util.SafeCloseable;
 
-final class RequestContextAwareCompletableFuture<T> extends CompletableFuture<T> {
+final class Java9RequestContextAwareCompletableFuture<T> extends CompletableFuture<T> {
```

Comment:
Could this class extend `RequestContextAwareCompletableFuture` to reduce duplication?

Reply:
The main reason not to extend is to avoid the complexity of `maybeContextAware`. I think since Java 8 API won't be added to, the duplication is ok since there's no maintenance burden. What do you think?

Reply:
SGTM